library/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="NaraeImagePickerTheme" parent="@style/Theme.AppCompat.Light.DarkActionBar"/>
```

Comment:
When applying a style in Activity by programmatically, you don't need ```NaraeImagePickerTheme``` and extending that style. I can produce the same effort by this statement.

```
<style name="CustomPickerTheme" parent="Theme.AppCompat">
        <item name="colorPrimary">@color/colorPrimary</item>
        <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
        <item name="colorAccent">@color/colorAccent</item>
    </style>
```